### PR TITLE
FIX Remove TopPageID from editable fields

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-elemental/issues/1041

### CODE REVIEW

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
The same treatment is needed in `FluentExtension` to hide the `TopPageLocale` field.

File: src/TopPage/DataExtension.php [hunk 1]
```diff
@@ -255,6 +255,11 @@ public function withFixedTopPage(int $topPageID, callable $callback)
         }
     }
 
+    public function updateCMSFields($fields)
```
Comment (GuySartorelli) [inline src/TopPage/DataExtension.php]:
```suggestion
    public function updateCMSFields(FieldList $fields)
```
Needs to match the superclass's method signature
https://github.com/silverstripe/silverstripe-framework/blob/66561ccb49e196bd652b1fc2112c5aeda062354c/src/ORM/DataExtension.php#LL218C36-L218C55

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: src/TopPage/DataExtension.php (hunk 1 above)
Comment (emteknetnz) [reply src/TopPage/DataExtension.php]: Updated

**Comment by emteknetnz** [comment]
Have updated FluentExtension - good spot

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**